kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/food2door/HealthyShop.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.good.patterns.challenges.food2door;
+
+import java.util.Map;
+
+public class HealthyShop implements FoodProducer {
+
+    private Map<Product, Integer> stock;
+
+    public HealthyShop(Map<Product, Integer> stock) {
+        this.stock = stock;
+    }
+
+    public boolean productsInStock(Map<Product, Integer> foodOrder) {
+        for (Map.Entry<Product, Integer> entry : foodOrder.entrySet()) {
```

Comment:
ładny PoC. Niestety  nie zdejmujesz tutaj niczego ze stocku. Zostawmy to tak jednak bo i tak to co zrobiłeś jest troszkę nadmiarowe :)